pages/blog/2023-summary.md

```diff
@@ -0,0 +1,169 @@
+/> 
+
+Out of over 1.6k successful emails sent over the last nine months, the December Edition was the most popular. The reason is that that was when we released the AsyncAPI v3, one of our huge accomplishments in 2023.
+
+<Figure
+  src="/img/posts/2023-summary/total-opens.webp"
+/> 
+
+And, of course, I'm not exempt from unsubscribers. Although the number might seem minimal to some, it still stings as a writer.
+
+<Figure
+  src="/img/posts/2023-summary/unsubscribed.webp"
+/> 
+
+Looking forward to sending the 2024 Editions!!🎉🎉
+
+### Google Analytics
+
+The pageviews on the AsyncAPI website increased significantly over the past year. Our major contributor to these metrics comes from docs, and we expect to see more in the coming year as people want to know more about the AsyncAPI v3.
+
+<Figure
+  src="/img/posts/2023-summary/google-analytics.webp"
+/> 
+
+We also lost 21 days' worth of data during October due to a PR update that affected the analytics workflow, but we are doing well.
+
+### Google Search Console
+
+Google Search Console is one of my favorite tools as it shows how much impact and interest we are gaining. Compared to 2022, we can see a surge in impressions and, slowly but surely, in clicks as well. 
+
+<Figure
+  src="/img/posts/2023-summary/google-console.webp"
+/> 
+
+### Github
+
+The number of contributors in the community keeps growing, from having over **1600** contributors in 2022 to **2320** contributors in 2023. 
+
+<Figure
+  src="/img/posts/2023-summary/Total-Contributors.webp"
+/> 
+
+Although we have a handful of maintainers, we try our best to manage, resolve, and close issues as much as we can. Additionally, we are focusing on growing the number of maintainers in the coming years.
+
+<Figure
+  src="/img/posts/2023-summary/Issue-History.webp"
+/> 
+
+### NPM
+
+Numbers don't lie, and yes, these are the total downloads of the top AsyncAPI packages hosted in the Node Package Manager. We can also see the adoption of AsyncAPI spec downloads increased from 5M to 17M since 2022.
+
+<Figure
+  src="/img/posts/2023-summary/NPM-packages.webp"
+/> 
+
+So, if you are worried about the project slowing down anytime soon, we are still growing strong.
+
+## Community Achievements
+
+We achieved so much this year, and here are some of the highlights from the AsyncAPI community in 2023!
+
+- We had our first community hire, [Thulisile Sibanda](https://www.linkedin.com/in/v-thulisile-sibanda/), to work full-time as the project's Community Manager.
+- [Lukasz Gornicki](https://twitter.com/derberq) went from Community Guardian to Executive Director (boss moves right there 😂😂). And yes, he's planning to get rid of the title soon.
+- The [Ambassador Program](https://www.asyncapi.com/community/ambassadors) increased this year. 
+- We launched the AsyncAPI Community Updates newsletter.
+- Released the [Parser-JS v2](https://github.com/asyncapi/parser-js).
+- Launched the AsyncAPI Bounty Program.
```

Comment:
maybe in one sentence you can mention how much issues we already resolved thanks to it, and link to PR with bounty rules?